feature-utils/poly-look/test/react-components/overlays/base.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import BaseOverlay from "../../../src/react-components/overlays/base";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+const testClassName = "test1";
+const testId = "testId";
+const testComponent = (
+  <BaseOverlay className={testClassName}>
+    <div data-testid={testId}></div>
+  </BaseOverlay>
+);
+
+const centeredTestComponent = (
+  <BaseOverlay className={testClassName} centered={true}></BaseOverlay>
+);
+
+it("renders correctly", () => {
+  const { container, getByTestId } = render(testComponent);
+  expect(container).toBeTruthy();
```

Comment:
You can probably skip this. It's my fault, I know, because I put it in the initial examples for this, but I was feeling my way around the library and it was all I could come up with initially. (Here, @fhd, I'm saving you a nit)

Reply:
Didn't put @fhd as reviewer as one of you should be enough for this

Reply:
Well, you can then expect me to be truthy about not needing toBeTruthy tests...